README.md

```diff
@@ -262,7 +262,6 @@ POSTGRES_PASSWORD=$(kubectl get secret artifactory-postgresql -o jsonpath="{.dat
 
 ```bash
 helm upgrade --install artifactory jfrog/artifactory \
-       --set artifactory.masterKey=$MASTER_KEY \
```

Comment:
Is this intended?

Reply:
Yes. There's no need providing MASTER_KEY when doing a helm upgrade (I'm slowly removing it from all repos). You only need it for a fresh install